workflows/iuf/operations/managed-nodes/managed-nodes-rollout.yaml

```diff
@@ -230,19 +230,21 @@ spec:
               fi
 
               if [ -z "$sat_command" ] ; then
-                  echo "ERROR: Failed to create a valid sat bootsys command."
+                  echo "ERROR Failed to create a valid sat bootsys command. It might be because managed_rollout_strategy is not set to either reboot or stage"
                   exit 1
               fi
 
+              echo "DEBUG sat command created is $sat_command"
```

Comment:
Because this relates to something the user may have understanding of, I would change this from DEBUG to ERROR since it may provide the user with a clue of what went wrong.